Makefile

```diff
@@ -413,6 +419,14 @@ release-prereqs: charts
 	@if [ $(CALICO_VER) != $(NODE_VER) ]; then \
 		echo "Expected CALICO_VER $(CALICO_VER) to equal NODE_VER $(NODE_VER)"; \
 		exit 1; fi
+	# Ensure that install-calico-windows.ps1 is the same version as the release
+	# version in node.
+	$(MAKE) update-install-calico-windows-script NODE_BRANCH_FOR_SCRIPT=$(NODE_VER)
```

Comment:
isn't `NODE_VER` here going to be of the form `X.Y.Z` instead of `release-X.Y` like our branches are usually named? Or is it OK that this is the node's tag?

Reply:
Looking at the rest of this, maybe we shouldn't call it `NODE_BRANCH` but something like `NODE_TAG` or `NODE_VER`. WDYT?

Reply:
We chatted about this on Slack and I added d8aa221a08bb1e15371cc186187bd0d82e743005